src/list/foundation.tsx

```diff
@@ -244,7 +246,11 @@ export const useListFoundation = (props: ListProps & React.HTMLProps<any>) => {
   // layout on mount
   useEffect(() => {
```

Comment:
I think each prop should have its own useEffect hook.

Originally, foundation.layout would run only when the foundation variable changed, which is actually never. Aka, it would only run once on mount.

I'm not actually certain what the other props actually do. It is possible you want layout to always run when they change as well, but I think you probably only want the foundation callbacks to to be called when the individual prop changes. You wouldn't want to call `setWrapFocus` when the vertical prop changed for instance.

Reply:
Hi @jamesmfriedman. I think you are right. The `foundation.layout()` method does not need to be recalled after `vertical or `wrapFocus` is set. I am not sure that my solution is ideal, insofar as handling default values. Is it correct that at present you do not have a way to handle default values in this project?

Reply:
Surprisingly, I don't think any default values are actually needed anywhere in the library, this might be one of the only cases.

Everything else looks great